src/components/Profile/components/PersonalInfoList/components/AlumniUpdateForm/index.js

```diff
@@ -0,0 +1,418 @@
+          break;
+        default:
+          break;
+      }
+    }
+    setDisableUpdateButton(hasError || !hasChanges);
+  }, [updatedInfo, currentInfo]);
+
+  const handleChange = (event) => {
+    const getNewInfo = (currentValue) => {
+      return {
+        ...currentValue,
+        [event.target.name]:
+          event.target.type === 'checkbox' ? event.target.checked : event.target.value,
+      };
+    };
+    setUpdatedInfo(getNewInfo);
+  };
+
+  const getFieldLabel = (fieldName) => {
+    const matchingField = allFields.find((field) => field.name === fieldName);
+    return matchingField.label;
+  };
+
+  function getUpdatedFields(updatedInfo, currentInfo) {
+    const updatedFields = [];
+    Object.entries(currentInfo).forEach(([field, value]) => {
+      let updatedValue = value;
+      if (field === 'homePhone' || field === 'workPhone' || field === 'mobilePhone') {
+        updatedValue = value.replace(/[-()\s]/g, '');
+      }
+      if (updatedInfo[field] !== value)
+        updatedFields.push({
+          Field: field,
+          Value: updatedValue,
+          Label: getFieldLabel(field),
+        });
+    });
+    return updatedFields;
+  }
+
+  const handleConfirm = () => {
+    setSaving(true);
+    let updateRequest = getUpdatedFields(currentInfo, updatedInfo);
+    updateRequest.push({
+      Field: 'changeReason',
+      Value: changeReason,
+      Label: 'Reason for change',
+    });
+    requestInfoUpdate(updateRequest).then(() => {
+      setSaving(false);
+      closeWithSnackbar({
+        type: 'success',
+        message: 'Your update request has been sent. Please check back later.',
+      });
+      handleWindowClose();
+    });
+  };
+
+  const handleWindowClose = () => {
+    setOpenConfirmWindow(false);
+    setChangeReason('');
+  };
+
+  const updateButton = isSaving ? (
+    <GordonLoader size={32} />
+  ) : (
+    <Button
+      variant="contained"
+      color="secondary"
+      onClick={() => {
+        setOpenConfirmWindow(true);
+      }}
+      disabled={disableUpdateButton}
+    >
+      Update
+    </Button>
+  );
+
+  /**
+   * @param {Array<{name: string, label: string, type: string, menuItems: string[]}>} fields array of objects defining the properties of the input field
+   * @returns JSX correct input for each field based on type
+   */
+  const mapFieldsToInputs = (fields) => {
+    return fields.map((field) => (
+      <ProfileUpdateField
+        error={field.error}
+        label={field.label}
+        name={field.name}
+        helperText={field.helperText}
+        value={updatedInfo[field.name]}
+        type={field.type}
+        menuItems={field.menuItems}
+        onChange={handleChange}
+      />
+    ));
+  };
+
+  return (
+    <>
+      <Card className={styles.update}>
+        <CardHeader
+          className={styles.update_title}
+          title="Update Information"
+          titleTypographyProps={{ variant: 'h4' }}
+        />
+        <CardContent>
```

Comment:
The dialog already handles everything that we used to need the Card for, so this can be removed.